fastify.js

```diff
@@ -760,10 +760,13 @@ function fastify (options) {
       const request = new Request(id, null, req, null, childLogger, onBadUrlContext)
       const reply = new Reply(res, request, childLogger)
 
+      const customMessage = 'incoming request'
       if (disableRequestLogging === false) {
-        childLogger.info({ req: request }, 'incoming request')
+        childLogger.info({ req: request }, createRequestLogMessage(customMessage))
       }
 
+      console.log('FST_ERR_BAD_URL(path): ', FST_ERR_BAD_URL(path))
+
```

Comment:
Ah yes, console.log an Error. Double performance bottleneck

```suggestion
```

Reply:
Ok I will make sure to review that later.  Thanks